dbms/src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -14,172 +21,166 @@ class ASTStorage;
-    /** Limit parallel fetches */                                                                             \
-    M(SettingUInt64, replicated_max_parallel_fetches, 0)                                                      \
-    M(SettingUInt64, replicated_max_parallel_fetches_for_table, 0)                                            \
-                                                                                                              \
-    /** Limit parallel fetches from endpoint (actually pool size) */                                          \
-    M(SettingUInt64, replicated_max_parallel_fetches_for_host, DEFAULT_COUNT_OF_HTTP_CONNECTIONS_PER_ENDPOINT)\
-                                                                                                              \
-    /** Limit parallel sends */                                                                               \
-    M(SettingUInt64, replicated_max_parallel_sends, 0)                                                        \
-    M(SettingUInt64, replicated_max_parallel_sends_for_table, 0)                                              \
-                                                                                                              \
-    /** If true, Replicated tables replicas on this node will try to acquire leadership. */                   \
-    M(SettingBool, replicated_can_become_leader, true)                                                        \
-                                                                                                              \
-    M(SettingSeconds, zookeeper_session_expiration_check_period, 60)                                          \
-                                                                                                              \
-    /** Check delay of replicas settings. */                                                                  \
-                                                                                                              \
-    /** Period to check replication delay and compare with other replicas. */                                 \
-    M(SettingUInt64, check_delay_period, 60)                                                                  \
-                                                                                                              \
-    /** Period to clean old queue logs, blocks hashes and parts */                                            \
-    M(SettingUInt64, cleanup_delay_period, 30)                                                                \
-    /** Add uniformly distributed value from 0 to x seconds to cleanup_delay_period                           \
+    /** How many rows correspond to one primary key value. */                                                     \
+    M(SettingUInt64, index_granularity, 8192, "")                                                                 \
```

Comment:
It's better to move comments to setting description for every setting.

Reply:
done